src/hock/ElementQueryHock.jsx

```diff
@@ -140,7 +144,9 @@ const ElementQueryDecorator = (eqs: ElementQuery[]): HockApplier => {
                     }
                 }
 
-                this.setState({width, height, active, inactive, ready: true});
+                if(this.mounted) {
```

Comment:
This should probably move up to the top of this method, there's no point computing all the active mqs if state is not going to be set